### PR TITLE
Remove GEO support (part 4) deleted IS_GEO & IS_GEO_POLY

### PR DESCRIPTION
**Pull request by sign60intel**

### CODE REVIEW

File: ImportExport/Importer.cpp [hunk 1]
```diff
@@ -1595,94 +1523,12 @@ void Importer::set_geo_physical_import_buffer(
     std::vector<int>& ring_sizes,
     std::vector<int>& poly_rings,
     int render_group) {
-  const auto col_ti = cd->columnType;
```
Comment (ienkovich) [inline ImportExport/Importer.cpp:1598]: In this patch, you remove the only usage of this method and therefore can remove this method.

File: ImportExport/Importer.cpp [hunk 2]
```diff
@@ -1595,94 +1523,12 @@ void Importer::set_geo_physical_import_buffer(
...
-    // Create render_group value and add it to the physical column
-    auto cd_render_group = catalog.getMetadataForColumn(cd->tableId, ++columnId);
-    TDatum td_render_group;
-    td_render_group.val.int_val = render_group;
-    td_render_group.is_null = is_null_geo;
-    import_buffers[col_idx++]->add_value(cd_render_group, td_render_group, false);
-  }
 }
 
 void Importer::set_geo_physical_import_buffer_columnar(
```
Comment (ienkovich) [inline ImportExport/Importer.cpp:1534]: I don't see any usages of this function. You can simply remove it.

File: ImportExport/Importer.cpp [hunk 3]
```diff
@@ -2769,17 +2230,7 @@ SQLTypes Detector::detect_sqltype(const std::string& str) {
 
     // then test for leading words
     if (str_upper_case.find("POINT") == 0) {
-      type = kPOINT;
```
Comment (ienkovich) [inline ImportExport/Importer.cpp:2772]: It's weird to leave the case for POINT and remove all others.

File: ImportExport/Importer.cpp [hunk 4]
```diff
@@ -2791,13 +2242,7 @@ SQLTypes Detector::detect_sqltype(const std::string& str) {
         // @TODO support eWKB, which has extra bits set in the geometry type
         auto first_five_bytes = str_upper_case.substr(0, 10);
         if (first_five_bytes == "0000000001" || first_five_bytes == "0101000000") {
-          type = kPOINT;
```
Comment (ienkovich) [inline ImportExport/Importer.cpp:2794]: And here too.

File: QueryEngine/NativeCodegen.cpp [hunk 5]
```diff
@@ -2035,9 +1945,6 @@ void Executor::createErrorCheckControlFlow(
             int32_t num_shift_by_blockDim = shared::getExpOfTwo(blockSize());
             int total_num_shift = num_shift_by_gridDim + num_shift_by_blockDim;
             uint64_t interrupt_checking_freq = 32;
-            auto freq_control_knob = g_running_query_interrupt_freq;
```
Comment (ienkovich) [inline QueryEngine/NativeCodegen.cpp:2038]: All modifications below in this file seem unrelated to GEO.

File: QueryEngine/TargetExprBuilder.cpp [hunk 6]
```diff
@@ -40,7 +40,7 @@ namespace {
 
 inline bool is_varlen_projection(const Analyzer::Expr* target_expr,
                                  const SQLTypeInfo& ti) {
-  return dynamic_cast<const Analyzer::GeoExpr*>(target_expr) && ti.get_type() == kPOINT;
+  return false;
```
Comment (ienkovich) [inline QueryEngine/TargetExprBuilder.cpp:43]: Do you make a list of functions to be later removed? It might be hard later to find them.

Comment (sign60intel) [reply QueryEngine/TargetExprBuilder.cpp:43]: > Do you make a list of functions to be later removed? It might be hard later to find them.

yes, I do this list

File: ImportExport/Importer.cpp (hunk 1 above)
Comment (sign60intel) [reply ImportExport/Importer.cpp:1598]: @ienkovich  fixed

File: ImportExport/Importer.cpp (hunk 2 above)
Comment (sign60intel) [reply ImportExport/Importer.cpp]: @ienkovich fixed

File: ImportExport/Importer.cpp (hunk 4 above)
Comment (sign60intel) [reply ImportExport/Importer.cpp:2794]: @ienkovich fixed

File: QueryEngine/NativeCodegen.cpp (hunk 5 above)
Comment (sign60intel) [reply QueryEngine/NativeCodegen.cpp:2038]: @ienkovich 
the definition of ```g_running_query_interrupt_freq``` is under ```ENABLE_GEOS`` macro in this file

File: ImportExport/Importer.cpp (hunk 3 above)
Comment (sign60intel) [reply ImportExport/Importer.cpp:2772]: @ienkovich  fixed

**Comment by sign60intel** [comment]
> ```
> $ grep GeoOps.cpp .. -rI
> ../QueryEngine/cuda_mapd_rt.cu:#include "GeoOps.cpp"
> ```

@ienkovich fixed

**Review by ienkovich — APPROVED** [review]
